apps/starwar-graphql/src/people/people.resolver.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import request = require('supertest');
+import { PeopleModule } from './people.module';
+
+const mockPeople = [
+  {
+    "id": "cGVvcGxlOjEz",
+    "name": "Chewbacca",
+    "species": {
+      "name": "Wookie",
+      "homeworld": {
+        "name": "Kashyyyk",
+        "diameter": 12765
+      }
+    }
+  },
+  {
+    "id": "cGVvcGxlOjE0",
+    "name": "Han Solo",
+    "species": null
+  }
+]
+const mockGetAllPeople = jest.fn(async () => mockPeople)
+
+jest.mock('../infrastructure/starWar.adapter', () => {
+  return {
+    StarWarAdapter: jest.fn().mockImplementation(() => {
+      return {getAllPeople: mockGetAllPeople};
+    }),
+  };
+});
+
+describe('[e2e] PeopleResolver', () => {
+  let app: INestApplication;
+
+  beforeEach(async () => {
+    mockGetAllPeople.mockClear();
+
+    const moduleRef: TestingModule = await Test.createTestingModule({
+      imports: [PeopleModule],
+    }).compile();
+
+    app = moduleRef.createNestApplication();
+    await app.init();
+  });
+
+  afterAll(async () => {
+    await app.close();
+  });
+
+  const gql = '/graphql';
+
+  describe('getAllPeople', () => {
```

Comment:
Try to demonstrate running a test case
------------
[e2e] PeopleResolver
	getAllPeople
	  should return id and name
	  should return id, species and homeworld name
------------

Let's see "should return" in 2 cases it is so confusing. why did 2 test cases not return the same result?

Reply:
Hint: change the description on your test case.